crates/bdk/src/wallet/tx_builder.rs

```diff
@@ -274,12 +276,20 @@ impl<'a, D, Cs: CoinSelectionAlgorithm, Ctx: TxBuilderContext> TxBuilder<'a, D,
     ///
     /// These have priority over the "unspendable" utxos, meaning that if a utxo is present both in
     /// the "utxos" and the "unspendable" list, it will be spent.
-    pub fn add_utxos(&mut self, outpoints: &[OutPoint]) -> Result<&mut Self, Error> {
+    pub fn add_utxos(
```

Comment:
Same here, if you want to manually add utxos you probably don't care whether they're unlocked or not. 

Unless... we might want to return a specific error (not `UnknownUtxo`, a more detailed one, like `LockedUtxo`) if you try to add a locked utxo to a txbuilder that doesn't have `include_reserved_utxos`. I'm not sure exactly what's the way forward here.

Reply:
I will prefer creating an error like `LockedUtxo`. Make more sense to me.